redis-consumer/Dockerfile

```diff
@@ -3,18 +3,10 @@ FROM python:2.7.11
 MAINTAINER Robert Kaye <rob@metabrainz.org>
 
 # General setup
-RUN apt-get update && apt-get install -y --no-install-recommends python-redis redis-tools vim
```

Comment:
Why do not need the PG client here? Is this for the command line tools only and sqlalchemy doesn't need this?